src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>Third Party Free Format Data</td>
+                </tr>
+                <tr>
+                    <td>1</td>
+                    <td>Reserved</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Bluetooth Record</td>
+                </tr>
+                <tr>
+                    <td>3-9</td>
+                    <td>Reserved</td>
+                </tr>
+                <tr>
+                    <td>10</td>
+                    <td>Driver ID</td>
+                </tr>
+                <tr>
+                    <td>11</td>
+                    <td>Curve Logging</td>
+                </tr>
+                <tr>
+                    <td>12</td>
+                    <td>Logging With Timestamp</td>
+                </tr>
+                <tr>
+                    <td>13</td>
+                    <td>Protobuf Data</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Type 0 Third Party Free Format Data</h4>
+        <p>The maximum size is 27 bytes. Rate limit is 500 logs per 10 minutes. If you exceed the rate limit, the GO device will stop taking data from the IOX.</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Driver ID</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-27</td>
+                    <td>Data</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Type 2 Bluetooth Record</h4>
+        <p>Rate limit is 1200 logs per 10 minutes. If you exceed the rate limit, the GO device will stop taking data from the IOX.</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Bluetooth Record</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-6</td>
+                    <td>MAC Address</td>
+                </tr>
+                <tr>
+                    <td>1</td>
+                    <td>Data Type</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>FP24 Value</td>
+                </tr>
+            </tbody>
+        </table>
+        <p>Further details can be found here: <a href="/sdk/hardware/addon-protocols/ble/">Add-On Protocol - BLE</a></p>
+        <h4>Type 11 Curve Logging</h4>
+        <p>This message can be used to send the 4-byte (int32_t) data that is curve logged by the GO. Additional information about curve logging can be found here: <a href="https://github.com/Geotab/curve" target="_blank" rel="noreferrer">Curve Logging</a></p>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on